### DISCUSSION TITLE
Is there a way to speed up the initial simulation when running forceDirectedGraph?

### DISCUSSION THREAD

**Discussion by RyanC10** [sgratzl/chartjs-chart-graph#68] [Ideas]
Is there a way to speed up the initial simulation when running forceDirectedGraph?

**Comment by sgratzl** [comment]
there is a simulation option called "initialIterations" 

https://github.com/sgratzl/chartjs-chart-graph/blob/5f0a4a6fb6418b8e1aaa00f25c4dfa8e7f35173e/src/controllers/ForceDirectedGraphController.ts#L40-L49
